index.js

```diff
@@ -14,7 +16,10 @@ const replacer = (match, group) => {
 
 class WhiteOut extends Stream.Transform {
   constructor (rules, options) {
-    super(Object.assign({}, options, { objectMode: true }));
+    options = options || {};
```

Comment:
This line and the `Object.assign()` could probably be combined if you add an assertion on `objectMode` being `true`. Up to you though.